SPECS/ostree/ostree.spec

```diff
@@ -123,16 +129,16 @@ install -vdm 755 %{buildroot}%{_sysconfdir}/ostree/remotes.d
 %{_unitdir}/ostree*.service
 %{_unitdir}/ostree-finalize-staged.path
 %{_libdir}/dracut/modules.d/98ostree/*
-%{_mandir}/man1/ostree-admin*
+%{_mandir}/man*/*.gz
```

Comment:
Should we be slightly more specific here? This seems like a really broad wildcard. Does this claim ownership of more files than it should?